src/services/kratos/index.ts

```diff
@@ -0,0 +1,296 @@
+import { getKratosConfig } from "@config"
+import {
+  AuthenticationKratosError,
+  KratosError,
+  LikelyNoUserWithThisPhoneExistError,
+  MissingTotpKratosError,
+  UnknownKratosError,
+} from "@domain/authentication/errors"
+import {
+  Configuration,
+  SuccessfulSelfServiceLoginWithoutBrowser,
+  V0alpha2Api,
+  V0alpha2ApiInterface,
+} from "@ory/client"
+import { authenticator } from "otplib"
+
+import { baseLogger } from "@services/logger"
+import { AxiosResponse } from "node_modules/@ory/client/node_modules/axios/index"
+
+export * from "./auth-phone-no-password"
+
+const { publicApi, adminApi } = getKratosConfig()
+
+const KratosSdk: (kratosEndpoint: string) => V0alpha2ApiInterface = (kratosEndpoint) =>
+  new V0alpha2Api(new Configuration({ basePath: kratosEndpoint }))
+
+export const kratosPublic = KratosSdk(publicApi)
```

Comment:
this must be module private. If we are leaking the `kratos` client outside of this module we are not implementing the `service` correctly.